### DISCUSSION TITLE
Firefly III - 500 Internal Server Error :(

### DISCUSSION THREAD

**Discussion by ardwk** [firefly-iii/firefly-iii#9520] [Firefly III]
### Support guidelines

- [x] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [x] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [x] ... there are no open or closed issues that are related to my problem
- [x] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

I use Cloudhosting, my funds use softaculous

### Debug information

Firefly III - 500 Internal Server Error :(
Whoops! An error occurred.
Unfortunately, this error was not recoverable :(. Firefly III broke. The error is:

Could not poll the database: could not find driver (Connection: mysql, SQL: select `id`, `name`, `data` from `configuration` where `name` = is_demo_site and `configuration`.`deleted_at` is null limit 1)

This error occurred in file /home/wkmediac/erp.wk-media.com/app/Support/FireflyConfig.php on line 65 with code 0.

More information
Please collect more information in the storage/logs directory where you will find log files. If you're running Docker, use docker logs -f [container]. You can read more about collecting error information in [the FAQ](https://docs.firefly-iii.org/how-to/general/debug/).

Get help on GitHub
You're more than welcome to open a new issue <strong><a href="https://github.com/firefly-iii/firefly-iii/issues">on GitHub</a></strong>. |raw

Use the search!
Include the information <a href="https://erp.wk-media.com/public/debug">from this debug page</a>.
Tell us more than "it says Whoops!"
Include error logs (see above).
Tell us what you were doing.

**Comment by JC5** [comment]
Hi, thanks for opening an issue. It looks like this isn't a bug. Rather, it seems you're missing the mysql driver in your PHP installation.

**Reply by ardwk** [reply]
I use cPanel on cloud hosting.
and when installing Firefly III I use Softaculous
